src/cards/MediaPlayerCard.js

```diff
@@ -17,6 +17,7 @@ class MediaPlayerCard extends AbstractCard {
    */
   #defaultOptions = {
     type: "custom:mushroom-media-player-card",
+    icon: undefined,
```

Comment:
Why added, the docs says it's optional or else why `undefined`?

Reply:
If it is not added, the icon used is set by the abstract card which is `mdi:help-circle`. 
When it is set to undefined, the icon used is the entity id's icon.